lib/Agrammon/Web/Routes.pm6

```diff
@@ -66,6 +66,21 @@ sub static-content($root) {
 
 sub api-routes (Str $schema, $ws) {
     openapi $schema.IO, {
+        # working
+        operation 'createAccount', -> LoggedIn $user {
+            request-body -> (:$email!, :$password!, :$key, :$firstname, :$lastname, :$org) {
+                my $username = $ws.create-account($user, $email, $password, $key, $firstname, $lastname, $org);
+                content 'application/json', { :$username };
+                CATCH {
+                    note "$_";
+                    when X::Agrammon::DB::User::AlreadyExists | X::Agrammon::DB::User::CreateFailed {
+                        conflict 'application/json', %(
+                            error => .message
```

Comment:
That the bug above isn't caught means we don't have a test covering this line.

Reply:
Added test.

Reply:
With the current setup I don't see that `X::Agrammon::DB::User::CreateFailed` could be thrown without a DB-failure.
I left it in there anyway and added a test for it in `user.t`.

Reply:
Whether all of these should really be 409 is another question, though... I mean, already exists is obviously a conflict. But "unknown role" doesn't really feel like one. It's good we're using a HTTP error code, but perhaps we should try and pick appropriate ones.